viewer/src/models/cad/CadLoadingHints.ts

```diff
@@ -14,5 +14,5 @@ export type CadLoadingHints = {
 };
 
 export const defaultLoadingHints: Required<CadLoadingHints> = {
-  maxQuadSize: 0.0025
+  maxQuadSize: 0.025
```

Comment:
Is this intended to change in this PR? This will change how all models load.

Reply:
OOoooooOOoooooOOOps 2